src/onegov/feriennet/forms/notification_template.py

```diff
@@ -54,6 +55,18 @@ def validate_subject(self, field: StringField) -> None:
                 _("A notification with this subject exists already")
             )
 
+    def process_obj(
+        self,
+        model: NotificationTemplate  # type:ignore[override]
+    ) -> None:
+
+        self.subject.data = Markup.escape(model.subject)
+        self.text.data = Markup.escape(model.text)
+
+    def populate_obj(self, model: NotificationTemplate) -> None:
+        model.subject = Markup.unescape(self.subject.data)
+        model.text = Markup.unescape(self.text.data)
+
```

Comment:
```suggestion
```
This change makes no sense to me. `def render` is still the correct place for the `escape`. You just need to pass `Markup(self.text)` instead of `self.text` in the view that calls the `render` function. Eventually we may want to add a `Markup` column type, so we don't need to do this conversion for columns, that are supposed to store HTML content.

Reply:
Ok, done. But now getting again `Markup is used in a dangerous way`

Reply:
That's a flake8 plugin and it's correct that it would be dangerous, you can add a `noqa: MS001` comment to silence the error.

Reply:
And why would I need `variables.render(Markup(self.subject))` in my view as `render` takes a string returning Markup?

Reply:
Because otherwise the markup will be escaped by `escape`, wrapping in `Markup` prevents that. `escape(Markup("<br>")) == "<br>"`, whereas `escape("<br>") == "&lt;br&gt;`

Reply:
You could also get rid of the `escape` and change the signature to `Markup -> Markup` and pass `escape(self.subject)` and `Markup(self.text)`, if that is easier to understand for you.

Reply:
I see you haven't even restored the `escape` yet, so you can either do that or change the signature. It's up to you.

Reply:
I did `escape` when returning but figured out that its not doing what we expect as `expand_storage_links` returns Markup

```
def render(self, text: str) -> Markup:
        ...
        result = self.expand_storage_links(text)
        return Markup.escape(result)
```
I will follow your suggestion and change the signature to `Markup -> Markup`

Reply:
The escape needed to happen at the start, just like before, but you can also move it outside the function by changing the signature, yes.